dspace/config/workflow-curation.xml

```diff
@@ -61,6 +61,9 @@
        </task>
      </flowstep>
    </taskset>
```

Comment:
Based on the tests in this PR, we might want to improve the comments/docs for the `workflow-curation.xml`.  The comment at the beginning of this config file still references "DSpace provides three workflow steps" and tells you how to configure `workflow-curation.xml` based on those three steps.  It looks like that behavior has changed now.  Now, you need to know the Step "name" in `workflow.xml` and match it up in `workflow-curation.xml`... so, we should better document that in this config file.

Reply:
Argh, another place that I meant to revisit.  The documentation in the `.xsd` was closer but still referred to the old 3-step workflow model (since the text was initially copied from `workflow-curation.xml`).  Both should be more accurate and less murky now.